### PR TITLE
Typing in time when creating an event - Issue #656

### PR DESCRIPTION
**Pull request by Karina-Agliullova**
**Issue:** The user cannot modify time by typing it in.
**Solution:** I removed the "readonly" attribute to let the user edit the time if they want to type it in. Also, I changed the time format from 24-hour format to 12-hour format so that the user doesn't have to manually move to the other side when choosing time.
**Testing:** Tested by creating an event and typing in the time I wanted. 
**Issue:** https://github.com/BCStudentSoftwareDevTeam/celts/issues/656

### CODE REVIEW

File: app/templates/admin/createEvent.html [hunk 1]
```diff
@@ -129,7 +129,7 @@ <h1 id="title">{{page_title}}</h1>
                 {%else%}
                   {%set end_time = "12:00" %}
                 {% endif %}
-                  <input autocomplete="off" type="time" class="form-control timepicker readonly" value="{{end_time}}" name="timeEnd" aria-describedby="timeIcon2" placeholder="Pick an end time" onchange="update(this)" id=endTime required />
+                  <input autocomplete="off" type="time" class="form-control timepicker" value="{{end_time}}" name="timeEnd" aria-describedby="timeIcon2" placeholder="Pick an end time" onchange="update(this)" id=endTime required />
```
Comment (AndersonStettner) [inline app/templates/admin/createEvent.html:132]: Nice, this works but, if a user went to type something like 1:30, they would have to manually move over to the minute section with either a mouse click or a right arrow key. It would be nice to catch the : and move to the next input (the minutes) so the user doesn't have to manually move themselves.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]

File: app/templates/admin/createEvent.html (hunk 1 above)
Comment (Karina-Agliullova) [reply app/templates/admin/createEvent.html:132]: That's right. But I am a little confused. The user has to input two digits on the left and on the right of a colon symbol (:). That's why it doesn't let the user input 1:30, it has to be 01:30. Wouldn't removing the option of choosing two digits on the left result in the user not being able to choose time after 9, meaning 10:00, 11:00, and so on?

File: app/static/js/createEvents.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ $(document).ready(function() {
   // everything except Chrome
   if (navigator.userAgent.indexOf("Chrome") == -1) {
     $('input.timepicker').timepicker({
-             timeFormat : 'hh:mm p',
+             timeFormat : 'h:mm p',
```
Comment (AndersonStettner) [inline app/static/js/createEvents.js]: This does not appear to have any effect. What was this change intended to do?

**Review by AndersonStettner — CHANGES_REQUESTED** [review]

File: app/static/js/createEvents.js (hunk 2 above)
Comment (Karina-Agliullova) [reply app/static/js/createEvents.js]: This change lets you type in only one digit except when you start with 1 because it can be 10,11,12. Basically, I switched from 24-hour format to 12-hour format so that when you press anything more than 1, it will automatically take you to the minutes, so you don't have to press any keys or do anything manually

**Review by AndersonStettner — APPROVED** [review]
Looks good!
